model/CHANGELOG.md

```diff
@@ -7,4 +7,5 @@ This file documents changes to the `rawls-model` library, including notes on how
 Added:
 - Support 2.13
 
-SBT dependency: `"org.broadinstitute.dsde" %% "rawls-model" % "0.1-2356e282"`
+SBT dependency: `"org.broadinstitute.dsde" %% "rawls-model" % "0.1-e55b488f-SNAP"`
+todo: don't forget to update this when we merge v2_workspaces
```

Comment:
Is there a Jira ticket for this?

Reply:
created https://broadworkbench.atlassian.net/browse/CA-1362 !